controllers/articles-controller.js

```diff
@@ -0,0 +1,24 @@
+const { selectTopics } = require("../models/articles-model")
+
+function getTopics(req, res, next) {
+    selectTopics(req.query)
+        .then((topics) => {
+            res.status(200).send(topics);
+        }).catch((err) => {
+            next(err);
+        });
+};
+
```

Comment:
Good implementation of catch. Make sure you're only passing req.query in when we're using a query.